src/php/tests/unit_tests/ServerCallTest.php

```diff
@@ -0,0 +1,268 @@
+<?php
+/*
+ *
+ * Copyright 2015 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+require_once(dirname(__FILE__) . '/../../lib/Grpc/ServerCallReader.php');
+require_once(dirname(__FILE__) . '/../../lib/Grpc/ServerCallWriter.php');
+require_once(dirname(__FILE__) . '/../../lib/Grpc/Status.php');
+
+// load protobuf from third_party
+set_include_path(get_include_path() . PATH_SEPARATOR . dirname(__FILE__) . '/../../../../third_party/protobuf/php/src/');
+
+spl_autoload_register(function ($className) {
+$classPath = str_replace('\\', DIRECTORY_SEPARATOR, $className);
+if (strpos($classPath, 'Google/Protobuf') === 0 || strpos($classPath, 'GPBMetadata/Google/Protobuf') === 0) {
+require_once($classPath . '.php');
```

Comment:
I think it's OK to leave these in the `_Test.php` code, but I'd like to avoid references to protobuf in the library code itself.